src/main/java/core/basesyntax/service/FileService.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.service;
+
+import java.util.List;
+
+public interface FileService {
+    List<String> readFromFile(String fromFileName);
+
+    void writeToFile(String toWriteFile, String toWriteData);

```

Comment:
Better move these methods to the separate classes

Reply:
done